### DISCUSSION TITLE
JSON examples

### DISCUSSION THREAD

**Discussion by kulnor** [CMSgov/price-transparency-guide#540] [General]
Greetings:
I'm doing some R&D work around APIs specifications and implementation to strengthen the use and adoption of the JSON schemas provided here. I'm looking for concrete examples and is there a list or registry of hospitals / providers publishing pricing data using machine readable formats (JSON or possibly XML)? Basically looking for public URLs where such files can be found (or sites to crawl for robots.txt).
many thanks

**Comment by escoses100** [comment]
This appears to be the best single source I have found - both the database and list of URLs - https://www.dolthub.com/repositories/dolthub/hospital-price-transparency-v3

**Reply by kulnor** [reply]
Thanks for the feedback. I've been in touch with dolthub but this database is more from the Transparency in Price and mostly points to  Excel or similar files.

**Comment by kulnor** [comment]
This page seem to be a useful starting point:  https://www.redcross.org/faq/transparency-in-coverage.html
